src/main/kotlin/no/nav/syfo/personstatus/db/createOrUpdatePersonOversiktStatus.kt

```diff
@@ -113,6 +113,7 @@ fun Connection.createPersonOversiktStatus(
         it.setBoolean(31, personOversiktStatus.trengerOppfolging)
         it.setObject(32, personOversiktStatus.trengerOppfolgingFrist)
         it.setBoolean(33, personOversiktStatus.behandlerBerOmBistandUbehandlet)
+        it.setObject(34, personOversiktStatus.latestOppfolgingstilfelle?.antallSykedager)
```

Comment:
Funker sikkert dette også, men kunne man bare gjort `setInt` her eller funker ikke det når den er nullable? (må i hvert fall sørge for at `null` ikke blir til 0 i basen). Samme gjelder i update under.

Reply:
Her er jo consumeren ajour (alle nye records på Kafka-topic'en har dette feltet) , så vi trenger kanskje ikke gardere mot null. Så foreslår it.setInt(...)